src/pages/Room/Room.js

```diff
@@ -62,13 +69,10 @@ class Room extends Component {
   };
 
   handleShow = () => {
-    this.setState(prevState => ({
-      show: true
-    }));
-    this.socket.emit('show', true);
+    this.socket.emit('show');
```

Comment:
Is there any particular reasons behind this change? I would like to keep the ability to set the value to either `true` or `false`, instead of only being able to set it to `true`.